Emily s dev

estephanian opened the pull request.

estephanian committed:
Create UsersTable.sql

users table with mock data inserted

estephanian committed:
Update UsersTable.sql

estephanian committed:
Create compfitmockdata.sql

mock data first submission

estephanian committed:
Update compfitmockdata.sql

estephanian committed:
Update compfitmockdata.sql

estephanian committed:
Create RegisterBackend.php

first commit

estephanian committed:
Update RegisterBackend.php

Tsiems commented on RegisterBackend.php:
```diff
@@ -0,0 +1,72 @@
+<?php
```
This PHP should be inside of the API folder in `index.php` to be run with an API call.

estephanian committed:
Create Create_New_User_Backend.php

estephanian committed:
Update Create_New_User_Backend.php

estephanian committed:
Create dependencies.php

estephanian committed:
Create routes.php

estephanian committed:
Create settings.php

estephanian committed:
Create teams.sql

estephanian committed:
Update teams.sql

for issue 8, "Include Teams Table" -- sort of unclear if this was exactly what was wanted.
